apps/content/api/portal/fonts.py

```
@@ -0,0 +1,329 @@
+    response={
+        200: FontDetailOut,
+        400: NinjaErrorResponse[Literal["font_name_required"]] | NinjaErrorResponse[Literal["external_url_required"]],
+        404: NinjaErrorResponse[Literal["font_not_found"]],
+    },
+)
+@permission_required([permission_class(PermissionChoice.PORTAL_UPDATE_FONT)])
+def update_font_put(
+    request: Request,
+    font_slug: str,
+    data: Form[FontPutIn],
+    thumbnail: UploadedFile | None = File(None),
```

Ruff (0.15.20)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)